apps/challenges/urls.py

```diff
@@ -130,6 +130,11 @@
         views.get_or_update_challenge_phase_split,
         name="get_or_update_challenge_phase_split",
     ),
+    url(
+        r"^challenge/slack_actions/$",
+        views.slack_actions,
+        name="challenge_slack_actions",
+    ),
```

Comment:
An open endpoint for slack to send any action to (any click or any interaction will be sent here)